public/docs/dart/latest/guide/security.jade

```diff
@@ -1 +1,30 @@
-include ../../../_includes/_ts-temp
\ No newline at end of file
+extends ../../../ts/latest/guide/security.jade
+
+block includes
+  include ../_util-fns
+
+block toc
+  :marked
+    * [Reporting Vulnerabilities](#report-issues)
+    * [Best Practices](#best-practices)
+    * [Preventing Cross-Site Scripting (XSS)](#xss)
+    <!-- * [Trusting Safe Values](#bypass-security-apis) -->
+    <!-- * [HTTP-level Vulnerabilities](#http) -->
+    * [Auditing Angular Applications](#code-review)
+
+block html-sanitization
+  :marked
+    Angular recognizes the value as unsafe, and automatically sanitizes it. It removes the `<script>`
+    tag but keeps safe content, producing output like this:
+
+    > **Binding innerHTML**<br>
```

Comment:
The formatting on this makes it look like a mistake, although I do appreciate that text is much easier to update than a screenshot. Any way we can put a box around it and/or change the font?